platform/ext/target/nordic_nrf/nrf5340/spm_hal.c

```diff
@@ -17,11 +17,27 @@
 #include "mpu_armv8m_drv.h"
 #include "region_defs.h"
 #include "secure_utilities.h"
+#include "region.h"
 
 /* Get address of memory regions to configure MPU */
 extern const struct memory_region_limits memory_regions;
 
+#if defined(CONFIG_TFM_ENABLE_MEMORY_PROTECT)
 static struct mpu_armv8m_dev_t dev_mpu_s = { MPU_BASE };
+#endif
+
+#ifdef CONFIG_TFM_ENABLE_MEMORY_PROTECT
```

Comment:
You can merge these `#if` blocks into one.

Reply:
done thanks